lib/plugins/chai/schema-matcher.js

```diff
@@ -1,12 +1,15 @@
 const validators = require('../../validators');
 
 module.exports = (Assertion, options) => {
-  Assertion.addMethod('matchApiSchema', function addApiSchemaMethod(apiDefinitionsPath) {
+  Assertion.addMethod('matchApiSchema', function addApiSchemaMethod() {
```

Comment:
this would be a breaking change. i think you should revert that

Reply:
done